glide.lock

```diff
@@ -1063,7 +1063,7 @@ imports:
 - name: github.com/shurcooL/sanitized_anchor_name
   version: 10ef21a441db47d8b13ebcc5fd2310f636973c77
 - name: github.com/sirupsen/logrus
-  version: 89742aefa4b206dcf400792f3bd35b542998eb3b
+  version: e54a77765aca7bbdd8e56c1c54f60579968b2dc9
```

Comment:
This level doesn't match what kube uses.  Kube uses 89742aefa4b206dcf400792f3bd35b542998eb3b

Reply:
fixed.